model/src/test/kotlin/AdvisorResultContainerTest.kt

```diff
@@ -90,6 +92,22 @@ class AdvisorResultContainerTest : WordSpec() {
 
                 deserializedAdvisorResults shouldBe advisorResults
             }
+
+            "be deserialized with vulnerabilities in the initial format" {
+                val resultsFile = File("src/test/assets/advisor-result-initial.yml")
+
+                val result = resultsFile.readValue<OrtResult>()
+
+                result.advisor.shouldNotBeNull()
```

Comment:
Please prefer to use the infix notation.

Reply:
I see that there is a `shouldNotBeNull()` function supporting infix notation, but this one expects an additional code block for further tests. Is it possible to use infix notation if I only want to state that the object is not null without providing additional tests?

Reply:
Yes. The syntax is `result.advisor shouldNot beNull()`.

Reply:
Thanks. Changed.